lib/model/compendium.js

```diff
@@ -20,6 +20,7 @@ var timestamps = require('mongoose-timestamp');
 var Compendium = new mongoose.Schema({
   id: String,
   user: String,
+  journal: String,
```

Comment:
It would be great if `journal` would have a default, otherwise all the compendia that are in existing servers might have a problem.

Can the journal be `NULL` ?